flaskr/translations.py

```diff
@@ -0,0 +1,19 @@
+translations = {
```

Comment:
Change the keys in the dictionary to be lower_case and spaces should be underscores.